internal/app/triggerservicebinding_mock_test.go

```diff
@@ -0,0 +1,65 @@
+// Code generated by mockery v2.9.4. DO NOT EDIT.
```

Comment:
Typically these generated mocks in a `mocks` sub folder. Any reason this one isn't?

Reply:
No particular reason I can change it

Reply:
now I remember its because I made the interface internal but since its an internal package anyway no real need to